content/en/tutorials/ecs-ecr-container-app/index.md

```diff
@@ -0,0 +1,553 @@
+
+[Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/) allows users to push their software packaged inside containers into an AWS-managed registry. Using ECR, you can version, tag, and manage your image lifecycles independently of your application. ECR is tightly integrated with other AWS services, such as ECS, EKS, and Lambda, and allows you to deploy your container image to these services. We can configure our ECS tasks to pull container images from ECR repositories. You can also use ECS task definitions to specify container settings such as CPU and memory limits, environment variables, and networking configuration, thus enabling you to focus on building and deploying your containerized applications with ease.
+
+[LocalStack Pro](https://localstack.cloud/) supports the creation of ECR registries and repositories, as well as ECS clusters and tasks on your local machine. In this tutorial, we will demonstrate how to use it to set up an NGINX web server to serve a static website by deploying CloudFormation templates on a local AWS environment provisioned by LocalStack.
+
+## Prerequisites
+
+-   [LocalStack Pro](https://localstack.cloud/pricing/)  to emulate the AWS services (SNS, SQS, SES, etc) locally
+    -   Don’t worry, if you don’t have a subscription yet, you can just get a trial license for free.
+-   [awslocal](https://docs.localstack.cloud/integrations/aws-cli/#localstack-aws-cli-awslocal)
+-   [Docker](https://docker.io/)
+-   [`cURL`](https://curl.se/download.html)
+
+## Creating the Docker image
+
+To start setting up an NGINX web server on an ECS cluster, we will first create the Docker image, which can be further pushed to an ECR repository. Let us start by creating a `Dockerfile` for our NGINX web server. The `Dockerfile` will be used to build the Docker image for our NGINX web server.
+
+```dockerfile
+FROM nginx
+
+ENV foo=bar
+```
+
+The `Dockerfile` uses the official `nginx` image from Docker Hub, enabling us to serve the default index page. Before building our Docker image, we will start LocalStack and create an ECR repository to push our Docker image. Start LocalStack with the `LOCALSTACK_API_KEY` environment variable to use your LocalStack Pro API key.
+
+```bash
```

Comment:
Please use the `{{ command }}` shortcode to prevent people selecting the `$` when copy-pasting. This is applicable on every ```bash on this page.